GeorgianMemenator/Scenes/DogBreedsList/DogBreedsListView.swift

```diff
@@ -11,15 +11,17 @@ import Foundation
 
 struct DogBreedsListView: View {
     @ObservedObject var viewModel: DogBreedsListViewModel
+    
     var body: some View {
-        EmptyView()
-        
-        Image(systemName: "dog")
-    }
-}
+        List(viewModel.dogBreeds) { dogBreed in
+                Text(dogBreed.name)
```

Comment:
Once you add minAge and maxAge properties to structure you need to refine the this view. So instead of only displaying `Text`, please consider:

```swift
VStack {
    Text(dogBreed.name)
    Text("min age: \(dogBreed.minAge), max age: \(dogBreed.maxAge)")
}
```

Reply:
I think, I have fixed everything. Waiting for your response.